### ISSUE TITLE
XML exported file doesn't have line break 

### ISSUE THREAD

**Issue by ahmed-al-bermawy** [ACFBentveld/XML#18]
First of all, thank you for this plugin 
the current issue I'm facing right now is the XML file doesn't make break line after each item, for example, this is supposed to be the XML after exporting
```
<?xml version="1.0" encoding="UTF-8"?>
<items>
    <file>
        <name>file1</name>
        <type>pdf</type>
    </file>
    <file>
        <name>file2</name>
        <type>png</type>
    </file>
</items>
```
but it appears like this 
```
<?xml version="1.0" encoding="UTF-8"?>
<items><file><name>file1</name><type>pdf</type></file><file><name>file2</name><type>png</type></file><file><name>file3</name><type>xml</type></file></items>
```
So it there any function that can handel this beak line
Thanks in advance

**Comment by AmandoVledder** [comment]
Hi @ahmed-al-bermawy 

Thanks for pointing out this issue.

I've just released version 2.4 that adds this feature. The docs can be found [here](https://acfbentveld.github.io/XML/docs/export#pretty-output).

Let me know if this works for you

**Comment by ahmed-al-bermawy** [comment]
I have upgraded to version 2.4 and ->usePrettyOutput() is working fine 
Thanks a lot for your quick response
